### ISSUE TITLE
[Cacti 1.2.13] Maximum line with CDEF causes graph to stop plotting correctly

### ISSUE THREAD

**Issue by kaf3773** [Cacti/cacti#3848]
# Describe the bug

Maximum line with CDEF on interface traffic graphs causes it to stop plotting correctly.

We apply the below CDEF to an item in the graph template.

![image](https://user-images.githubusercontent.com/16928454/95361330-999b2700-089a-11eb-857e-225ea042c412.png)

Below are the items in the graph template

![image](https://user-images.githubusercontent.com/16928454/95361796-2f36b680-089b-11eb-98a3-485cfdf6c8e9.png)

Red line shows up on the graph but IN (green area) and OUT (blue line) drop suddenly after applying the template. 

![image](https://user-images.githubusercontent.com/16928454/95362469-20043880-089c-11eb-8e08-ba6038310f7b.png)

We use this in version 0.8.8c by following these instructions https://forums.cacti.net/viewtopic.php?p=161323
But potting it over the version 1.2.13 does not work right.

**Comment by TheWitness** [comment]
Can you please post the output of the RRDtool syntax for the broken graph.  Should be able to goto Console > Management > Graphs, edit the Graph and then put the Graph into Debug mode.  Post the output.

**Comment by kaf3773** [comment]
We resolved this issue.

The issue was with the fact that when we cloned the template the wrong data query data source items were being used.

It was using  this

![image](https://user-images.githubusercontent.com/16928454/95771830-bca64c00-0c89-11eb-9986-93e168a48d4d.png)

We corrected it to use the 64bit Counters

![image](https://user-images.githubusercontent.com/16928454/95771945-eeb7ae00-0c89-11eb-8f36-21f022fb803f.png)

@TheWitness, thanks very much though for your suggestion and attempt to help.
